.github/workflows/tests.yml

```diff
@@ -117,9 +117,6 @@ jobs:
         id: determine-target
```

Comment:
Should we remove the `Determine emulator target` step completely, since it we always use `google_apis` now?